spinta/units/helpers.py

```diff
@@ -154,10 +155,20 @@ def is_time_unit(unit: str) -> bool:
 def is_si_unit(unit: str) -> bool:
     return _si_unit_re.match(unit) is not None
 
+def is_property_list(unit):
+    if unit is None:
+        return None
+    temp_rel = unit.split('[')
+    temp = temp_rel[-1].split(',')
+    temp = ''.join(temp).replace(']', '').split()
+    if isinstance(temp, list):
+        return True, temp, temp_rel[0]
```

Comment:
This is specific to tabular manifests and should go to `spinta/manifests/tabular/helpers.py`.